tests/sentry/tempest/endpoints/test_tempest_credentials.py

```diff
@@ -34,3 +37,73 @@ def test_client_secret_is_obfuscated(self):
 
     def test_unauthenticated_user_cant_access_endpoint(self):
         self.get_error_response(self.project.organization.slug, self.project.slug)
+
+    def test_create_tempest_credentials(self):
+        with Feature({"organizations:tempest-access": True}):
+            self.login_as(self.user)
+            response = self.get_success_response(
+                self.project.organization.slug,
+                self.project.slug,
+                method="POST",
+                **self.valid_credentials_data,
+            )
+            assert response.status_code == 201
+            creds_obj = TempestCredentials.objects.get(project=self.project)
+            assert creds_obj.client_id == self.valid_credentials_data["clientId"]
+            assert creds_obj.client_secret == self.valid_credentials_data["clientSecret"]
+            assert creds_obj.project == self.project
+            assert creds_obj.created_by_id == self.user.id
+
+    def test_create_tempest_credentials_without_feature_flag(self):
+        self.login_as(self.user)
+        response = self.get_error_response(
+            self.project.organization.slug,
+            self.project.slug,
+            method="POST",
+            **self.valid_credentials_data,
+        )
+        assert response.status_code == 404
```

Comment:
Would we want this to be a 403 Forbidden, right? 

from [mdn web docs](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/403): 
```
The HTTP 403 Forbidden [client error response](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status#client_error_responses) status code indicates that the server understood the request but refused to process it. This status is similar to [401](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/401), except that for 403 Forbidden responses, authenticating or re-authenticating makes no difference. The request failure is tied to application logic, such as insufficient permissions to a resource or action.
```